janusgraph-dist/src/assembly/static/cassandra/conf/cassandra.yaml

```diff
@@ -12,16 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# JanusGraph updated following from the default
+# * cluster_name
+# * data_file_directories
+# * commitlog_directory
+# * saved_caches_directory
+
 # Cassandra storage config YAML 
 
 # NOTE:
-#   See https://wiki.apache.org/cassandra/StorageConfiguration for
+#   See http://wiki.apache.org/cassandra/StorageConfiguration for
```

Comment:
Why it changed to `http`? Also, both links are broken, maybe we should use the next link? https://wiki.apache.org/confluence/display/CASSANDRA2/StorageConfiguration

Reply:
same here